recipes/nativefiledialog/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+import os
+from conans import ConanFile, MSBuild, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class NativefiledialogConan(ConanFile):
+    name = "nativefiledialog"
+    license = "ZLIB"
```

Comment:
```suggestion
    license = "Zlib"
```

https://spdx.org/licenses/Zlib.html